assemble/bin/accumulo-cluster

```diff
@@ -184,14 +184,14 @@ isDebug() {
 }
 
 debug() {
-  isDebug && echo "${@@P}"
+  isDebug && echo "$1"
```

Comment:
I get that this is being done because it is assuming that there's only one arg because debugAndRun has been changed to flatten it to a single arg, but if this code is called anywhere else other than debugAndRun where that's not the case, then only the first arg is taken and the rest ignored and not printed. This could just be:

```suggestion
  isDebug && echo "$@"
```